seshdash/models.py

```diff
@@ -45,18 +45,27 @@ class Sesh_User(models.Model):
     #to be added
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     department = models.CharField(max_length=100)
+    def __str__(self):
+        return self.user.username
 
 
 class Alert_Rule(models.Model):
+    OPERATOR_CHOICES = (
+        ("eq" , "equals"),
+        ("lt" , "less than"),
+        ("gt" , "greater than"),
+        )
     site = models.ForeignKey(Sesh_Site)
-    users = models.ForeignKey(Sesh_User)
-    field1 = models.CharField(max_length=100)
-    value1 = models.FloatField()
-    operator1 = models.IntegerField() #0 equals , 1 less than, 2 greater than
-    field2 = models.CharField(max_length=100)
-    value2 = models.FloatField()
-    operator2 =  models.IntegerField()
+    check_field = models.CharField(max_length=100)
+    value = models.FloatField()
+    operator = models.CharField(max_length=2,
+                                      choices=OPERATOR_CHOICES,
+                                      default="lt")
+    send_mail = models.BooleanField(default=True)
+    #TODO a slug field with the field operator and value info can be added
     #TODO this is vastly incomplete!! fields need to be mapable and chooices need to exist
+    def __str__(self):
+        return "site:%s rule:[%s '%s' %s]" %(self.site.site_name,self.check_field,self.operator,self.value)
 
 #TODO Add alert Object to save alerts
 class Sesh_Alert(models.Model):
```

Comment:
You have implemented Sesh_Alert model before. I just used it as is. 

```python
class Sesh_Alert(models.Model):
    site = models.ForeignKey(Sesh_Site)
    alert = models.ForeignKey(Alert_Rule)
    date = models.DateTimeField()
    isSilence = models.BooleanField()
    alertSent = models.BooleanField()
```

Currently when a rule trigger an alert a Sesh_Alert object is created in alert.py Line 27
```python
            ...
            alert_obj = Sesh_Alert.objects.create(site = data_point.site, alert=rule, date=timezone.now(),
                                                  isSilence=True,alertSent=rule.send_mail)
            if rule.send_mail:
                mail_sent = alert(data_point,content,recipients)
                alert_obj.alertSent = mail_sent
                print("Sent mail for %s" %content)
            alert_obj.save()
```